sympy/logic/boolalg.py

```diff
@@ -2782,6 +2793,16 @@ def simplify_logic(expr, form=None, deep=True, force=False):
     if form not in (None, 'cnf', 'dnf'):
         raise ValueError("form can be cnf or dnf only")
     expr = sympify(expr)
+    # check for quick exit: right form and all args are
```

Comment:
reviewer: please check to see that this is the right logic for quick exit: if an expression is in the right form and all args are literal then there is nothing left to do.